src/Conference.tsx

```diff
@@ -322,16 +322,22 @@ const Conference: React.FC = (props) => {
 
   useEffect(() => {
     bandwidthRtc.onStreamAvailable((stream: RtcStream) => {
-      setRemoteStreams({
-        ...remoteStreams,
-        [stream.endpointId]: stream,
+      console.log("onStreamAvailable", stream);
+      setRemoteStreams(currentStreams => {
```

Comment:
Turns out you have to use the previous state passed to you by `setState` if you don't want to have a Bad Time with race conditions.